### PR TITLE
network: compiles

### PR DESCRIPTION
**Pull request by mvidner**
note that there's also a patch adding a missing include, which is a bug in
yast-network master:
https://github.com/yast/yast-network/pull/75#issuecomment-16511001

### CODE REVIEW

File: patches/network.patch [hunk 1]
```diff
@@ -0,0 +1,124 @@
+diff --git a/src/clients/inst_netprobe.ycp b/src/clients/inst_netprobe.ycp
+index b0609ea..fcb2c5a 100644
+--- a/src/clients/inst_netprobe.ycp
++++ b/src/clients/inst_netprobe.ycp
+@@ -33,6 +33,8 @@ textdomain "network";
+ 
+ import "NetHwDetection";
+ 
++include "network/routines.ycp";
```
Comment (jreidinger) [inline patches/network.patch:11]: strange, why you need it?

File: patches/network.patch [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
++
+    case `edit:
+ 	if( LanItems::IsCurrentConfigured())
+         {
+@@ -355,7 +357,7 @@ If you edit the settings for this interface here,
+ the interface will no longer be managed by NetworkManager.
+ ")))
+                 {
+-                    break; // means cancel
++                    return nil; // means cancel
```
Comment (jreidinger) [inline patches/network.patch:51]: why this change is needed?

File: patches/network.patch (hunk 1 above)
Comment (mvidner) [inline patches/network.patch:11]: because of a recent bug in upstream, see the commit message

File: patches/network.patch (hunk 2 above)
Comment (mvidner) [inline patches/network.patch:51]: Because `break` can be used not only to _end_ a `case` but also to break out of the middle of the `switch`. Ruby cannot do that and y2r cries "Misplaced break". That would be difficult to translate in general but fortunately `return` from the whole function can be used here. I'll add a comment:

```
// y2r: cannot break from middle of switch but in this function return will do
```

Comment (jreidinger) [inline patches/network.patch:51]: yes, comments are nice to have there

File: patches/network.patch (hunk 1 above)
Comment (jreidinger) [inline patches/network.patch:11]: Ah, OK, maybe add TODO as I think it will be removed quite quickly

**Comment by mvidner** [comment]
Updated.
